### PR TITLE
#14: Fixed bug where iOS would return true even if network was not in…

### LINKED ISSUE #14: iOS returns true even if the network was not found
NEHotspotConfigurationManager.shared.apply(hotspotConfiguration) 

finished without an error. We should check if the user is connected to that specific ssid. in the completion handler.

### PR DESCRIPTION
**Pull request by vanlooverenkoen**
Fixed #14

### CODE REVIEW

File: ios/Classes/SwiftWifiConnectorPlugin.swift [hunk 1]
```diff
@@ -4,6 +4,18 @@ import NetworkExtension
 import SystemConfiguration.CaptiveNetwork
 
 public class SwiftWifiConnectorPlugin: NSObject, FlutterPlugin {
+        
+  var currentWifiSSD: String? {
```
[the remark this answers is not in the document]
Comment (vanlooverenkoen) [reply ios/Classes/SwiftWifiConnectorPlugin.swift:8]: Everytime after a connection so is can/will change overtime

[the remark this answers is not in the document]
Comment (wonjerry) [reply ios/Classes/SwiftWifiConnectorPlugin.swift:8]: nice 👍

**Review by wonjerry — APPROVED** [review]

**Comment by fbuireu** [comment]
What's the status of this PR? I really need this changes on master

**Comment by vanlooverenkoen** [comment]
This is working perfectly. It should only be merged

**Comment by wonjerry** [comment]
I approved this pr, but you can't merge?  If so, I will

**Comment by vanlooverenkoen** [comment]
I have no write permission. But if possible It would be nice to have more access on this repository. That way I can not only manage open issues & review other pr's but also merge when possible.

**Comment by wonjerry** [comment]
I invited you as a collaborator. please confirm

**Pull request merged**